test/Handlebars.Net.Helpers.Tests/Templates/ObjectHelpersTemplateTests.cs

```diff
@@ -38,4 +39,65 @@ public void ToString_WithInt()
         // Assert
         result.Should().Be("123.456");
     }
+
+    [CulturedFact("en-us")]
+    public void Compare()
```

Comment:
You could make this a culturedtheory and provide all possible methods : GreaterThan, LessThan, Equal, .....